cache/in-memory/src/permission.rs

```diff
@@ -22,12 +22,28 @@
 //! let cache = InMemoryCache::builder().resource_types(resource_types).build();
 //! ```
 //!
+//! # Disabled Member Communication Caveats
+//!
+//! The permission calculator checks the [current system time] against when a
+//! given member had their [communication disabled until]. If a member's
```

Comment:
am i missing something or should this be `communication timed out until` instead

Reply:
This verbiage matches the name of the field (`communication_disabled_until`), "timed out" is just the text in the UI